### PR TITLE
Remove flake8

### PR DESCRIPTION
**Pull request by ogajduse**
### Problem Statement
flake8 is long gone from robottelo, see #12621. 

### Solution
Let's remove its traces in the repository.

### CODE REVIEW

**Review by jameerpathan111 — APPROVED** [review]

**Comment by ogajduse** [comment]
trigger: test-robottelo
pytest:

**Comment by ogajduse** [comment]
trigger: test-robottelo

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8087
Build Status: UNSTABLE
PRT Comment: pytest /opt/app-root/src/robottelo/tests/foreman -v -m build_sanity --external-logging
Test Result : == 1 failed, 11 passed, 5492 deselected, 5602 warnings in 2164.27s (0:36:04) ===
```

File: docs/committing.rst [hunk 1]
```diff
@@ -41,11 +41,11 @@ documented, it doesn’t exist.
 In order to ensure you are able to pass the Travis CI build, it is recommended
 that you run the following commands in the base of your Robottelo directory.::
 
-    $ flake8 .
+    $ ruff check .
```
[the remark this answers is not in the document]
Comment (ogajduse) [reply docs/committing.rst:44]: I added ruff to requirements-optional.txt.

**Comment by ogajduse** [comment]
PRT failure is unrelated to the patch. Removing the `PRT-Failed` tag

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**